spec/std/isa/ext/Smepmp.yaml

```
@@ -0,0 +1,182 @@
+              ignored until a PMP reset, unless mseccfg.RLB is set.
+
+      b.  Adding a rule with executable privileges that either is M-mode-only or a locked Shared-
+          Region is not possible and such pmpcfg writes are ignored, leaving pmpcfg unchanged. This
+          restriction can be temporarily lifted by setting mseccfg.RLB e.g. during the boot process.
+
+      c.  Executing code with Machine mode privileges is only possible from memory regions with a
+          matching M-mode-only rule or a locked Shared-Region rule with executable privileges.
+          Executing code from a region without a matching rule or with a matching S/U-mode-only rule is
+          denied.
+
+      d.  If mseccfg.MML is not set, the combination of pmpcfg.RW=01 remains reserved for future standard use.
+
+      The truth table when the `mseccfg.MML` is set:
+
+      [cols="4*^.^1,2*^.^3", separator="!", %autowidth, options="header"]
+      !====
+      4+^! Bits on _pmpcfg_ register 2+^! Result
+      ! L ! R ! W ! X ! M Mode ! S/U Mode
+
+      ! 0 ! 0 ! 0 ! 0 2+^! Inaccessible region (Access Exception)
+      ! 0 ! 0 ! 0 ! 1 ! Access Exception ! Execute-only region
+      ! 0 ! 0 ! 1 ! 0 2+^! Shared data region: Read/write on M mode, Read-only on S/U mode
+      ! 0 ! 0 ! 1 ! 1 2+^! Shared data region: Read/write for both M and S/U mode
+      ! 0 ! 1 ! 0 ! 0 ! Access Exception ! Read-only region
+      ! 0 ! 1 ! 0 ! 1 ! Access Exception ! Read/Execute region
+      ! 0 ! 1 ! 1 ! 0 ! Access Exception ! Read/Write region
+      ! 0 ! 1 ! 1 ! 1 ! Access Exception ! Read/Write/Execute region
+      ! 1 ! 0 ! 0 ! 0 2+^! Locked inaccessible region* (Access Exception)
+      ! 1 ! 0 ! 0 ! 1 ! Locked Execute-only region* ! Access Exception
+      ! 1 ! 0 ! 1 ! 0 2+^! Locked Shared code region: Execute only on both M and S/U mode.*
+      ! 1 ! 0 ! 1 ! 1 2+^! Locked Shared code region: Execute only on S/U mode, read/execute on M mode.*
+      ! 1 ! 1 ! 0 ! 0 ! Locked Read-only region* ! Access Exception
+      ! 1 ! 1 ! 0 ! 1 ! Locked Read/Execute region* ! Access Exception
+      ! 1 ! 1 ! 1 ! 0 ! Locked Read/Write region* ! Access Exception
+      ! 1 ! 1 ! 1 ! 1 2+^! Locked Shared data region: Read only on both M and S/U mode.*
+      !====
+
+      *Locked rules cannot be removed or modified until a PMP reset, unless mseccfg.RLB is set.
+
+  Since all fields defined on mseccfg as part of this proposal are locked when set (MMWP/MML) or locked
+  when cleared (RLB), software can't poll them for determining the presence of Smepmp. It is expected
+  that BootROM will set mseccfg.MMWP and/or mseccfg.MML during early boot, before jumping to the
+  firmware, so that the firmware will be able to determine the presence of Smepmp by reading mseccfg
+  and checking the state of mseccfg.MMWP and mseccfg.MML.
+
+params:
+  MSECCFG_MML_TYPE:
+    schema:
+      enum: [read-only-0, read-only-1, sticky]
```

Examples in S.yaml look a bit different:

```suggestion
    schema:
      type: string
      enum: [read-only-0, read-only-1, sticky]
```
Try that modification, here and below.

---

It's done.